docs/docker-stack/entrypoint.rst

```diff
@@ -43,9 +43,9 @@ This can be achieved in various ways - you can change USER when you extend or cu
 you can dynamically pass the user to  ``docker run`` command, by adding ``--user`` flag in one of
 those formats (See `Docker Run reference <https://docs.docker.com/engine/reference/run/#user>`_ for details):
 
-```
-[ user | user:group | uid | uid:gid | user:gid | uid:group ]
-```
+.. code-block:: bash
+
+    [ user | user:group | uid | uid:gid | user:gid | uid:group ]
```

Comment:
And I don’t even know what this block is intended to be…

Reply:
hmmm.. I would say it's more "like" bash? 
https://docs.docker.com/engine/reference/run/#user

Reply:
Maybe just leave this unhighlighted? (using `text`)

Reply:
Sounds good. just updated